### ISSUE TITLE
The PDF is not accessible

### ISSUE THREAD

**Issue by markn86** [markn86/moodle-mod_customcert#302]
See https://moodle.org/mod/forum/discuss.php?d=389667.

**Comment by jauslander** [comment]
I never replied to the comment on the original thread so I will comment here. I tried to find someone to work on this for us but it did not work out. Adobe Acrobat has an accessibility check tool built in that allows users to check for document accessibility. When I run the tool on the generated PDF, it fails in a few different areas. I think the biggest issues are that there is no way to add alternate text to images and none of the content is tagged/labeled. If I use NVDA screen reader, none of the content is picked up.

**Comment by thejuliekramer** [comment]
I was wondering if Moodle had any plans to address this issue? We are also in need of accessible PDF generation. Thank you.

**Comment by mdjnelson** [comment]
You would have to create a tracker.moodle.org ticket and raise the issue before they would look at it. You should also while doing that create a forum post and get people to vote on the issue you created to give it some attention. This is not the correct location to discuss what Moodle HQ would do.

**Comment by mhughes2k** [comment]
Just to clarify this, is the issue that the (core) PDF Library that Moodle uses' doesn't support this, and so this plugin *can't* be fixed until that does?

I think the issue that we've identified this plugin doesn't provide a mechanism to enter alt-text or other "accessible" elements in it's own UI (which then of course need to get passed on to the PDF library itself for rendering/encoing/writing)
